docs/installation.md

```diff
@@ -204,8 +204,16 @@ java -XX:MinRAMPercentage=50 -XX:MaxRAMPercentage=80 \
 ### Helm <a name="helm"></a>
```

Comment:
This section is linked from docker.md - perhaps it should just link directly to the chart repo - wdyt?

Reply:
No .. I want to keep this since I want to actually get this onto a separate page and much improved.. so for now .. lets leave as is